CHANGELOG.md

```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+[1.5.0] - 2021-12-14
+--------------------
+
+### Changes
+
+- Support ansible-core 2.11 and 2.12
+
+### New features
+
+- Support specifying network connection via matching device path
+
+### Bug fixes
+
+- Fix the denial of setting the IPv6 DNS if the IPv6 family is not enabled
+- Fix the denial of setting the DNS search domain if no IP family is enabled
```

Comment:
What exactly is being fixed here for users? IIRC only the error messages became more clear. The actual bugfix related to this would be in https://github.com/linux-system-roles/network/pull/423

Reply:
ok, then I will drop "Bug fixes" for now.  Thanks.